src/locale/locales/nl/messages.po

```diff
@@ -0,0 +1,7903 @@
+msgid "Are you writing in <0>{0}</0>?"
+msgstr "Schrijft u in <0>{0}</0>?"
+
+#: src/screens/Onboarding/index.tsx:23
+#: src/screens/Onboarding/state.ts:82
+msgid "Art"
+msgstr "Kunst"
+
+#: src/view/com/composer/labels/LabelsBtn.tsx:170
+msgid "Artistic or non-erotic nudity."
+msgstr "Artistieke of niet-erotische naaktheid."
+
+#: src/screens/Signup/StepHandle.tsx:173
+msgid "At least 3 characters"
+msgstr "Minstens 3 tekens"
+
+#: src/components/dms/MessagesListHeader.tsx:75
+#: src/components/moderation/LabelsOnMeDialog.tsx:290
+#: src/components/moderation/LabelsOnMeDialog.tsx:291
+#: src/screens/Login/ChooseAccountForm.tsx:90
+#: src/screens/Login/ChooseAccountForm.tsx:95
+#: src/screens/Login/ForgotPasswordForm.tsx:123
+#: src/screens/Login/ForgotPasswordForm.tsx:129
+#: src/screens/Login/LoginForm.tsx:282
+#: src/screens/Login/LoginForm.tsx:288
+#: src/screens/Login/SetNewPasswordForm.tsx:154
+#: src/screens/Login/SetNewPasswordForm.tsx:160
+#: src/screens/Messages/components/ChatDisabled.tsx:133
+#: src/screens/Messages/components/ChatDisabled.tsx:134
+#: src/screens/Profile/Header/Shell.tsx:80
+#: src/screens/Signup/BackNextButtons.tsx:42
+#: src/screens/StarterPack/Wizard/index.tsx:307
+#: src/view/com/util/ViewHeader.tsx:87
+msgid "Back"
+msgstr "Terug"
+
+#: src/view/screens/Settings/index.tsx:442
+msgid "Basics"
+msgstr "Basis"
+
+#: src/components/dialogs/BirthDateSettings.tsx:106
+msgid "Birthday"
+msgstr "Verjaardag"
+
+#: src/view/screens/Settings/index.tsx:348
+msgid "Birthday:"
+msgstr "Verjaardag:"
+
+#: src/screens/Profile/Header/ProfileHeaderStandard.tsx:273
+#: src/view/com/profile/ProfileMenu.tsx:341
+msgid "Block"
+msgstr "Blokkeer"
+
+#: src/view/com/profile/ProfileMenu.tsx:280
+#: src/view/com/profile/ProfileMenu.tsx:287
+msgid "Block Account"
+msgstr "Blokkeer account"
+
+#: src/view/com/profile/ProfileMenu.tsx:324
+msgid "Block Account?"
+msgstr "Blokkeer account?"
+
+#: src/components/dms/ConvoMenu.tsx:188
+#: src/components/dms/ConvoMenu.tsx:192
+msgid "Block account"
+msgstr "Blokkeer account"
+
+#: src/view/screens/ProfileList.tsx:643
+msgid "Block accounts"
+msgstr "Blokkeer accounts"
+
+#: src/view/screens/ProfileList.tsx:747
+msgid "Block list"
+msgstr "Blokkeerlijst"
+
+#: src/view/screens/ProfileList.tsx:742
+msgid "Block these accounts?"
+msgstr "Blokkeer deze accounts?"
+
+#: src/view/com/util/post-embeds/QuoteEmbed.tsx:82
+msgid "Blocked"
+msgstr "Geblokkeerd"
+
+#: src/Navigation.tsx:149
+#: src/view/screens/ModerationBlockedAccounts.tsx:108
+msgid "Blocked Accounts"
+msgstr "Geblokkeerde accounts"
+
+#: src/screens/Moderation/index.tsx:280
+msgid "Blocked accounts"
+msgstr "Geblokkeerde accounts"
+
+#: src/view/com/profile/ProfileMenu.tsx:336
+msgid "Blocked accounts cannot reply in your threads, mention you, or otherwise interact with you."
+msgstr "Geblokkeerde accounts kunnen niet reageren op uw gesprekken, u niet vermelden en niet op een andere manier met u communiceren."
```

Comment:
```suggestion
msgstr "Geblokkeerde accounts kunnen niet reageren op uw gesprekken, u niet vermelden, en niet op een andere manier met u communiceren."
```

Reply:
Skipped; no comma before "en" in general when not ambiguous. Oxford comma in general not translated.